content/en/dashboards/ddsql_editor/reference/scalar_functions.md

```diff
@@ -0,0 +1,246 @@
+| Name | Return type | Description |
+|------|-------------|-------------|
+| array_to_string(array *a*, text *delimiter*, [text *nullString1*, *nullString2* ...]) | string | Concatenates array elements using the supplied *delimiter* and optional null strings (substrings to replace with `NULL`). |
+
+### unnest
+| Name | Return type | Description |
+|------|-------------|-------------|
+| unnest(array *a*) | variable | Returns each element in the array as a separate row. The return type is the element type of the array. `unnest` can only be used in the `SELECT` clause of a query. If other columns are selected with `unnest`, the value at each row in the table is repeated at each "output" row with each unnested element. If multiple columns are being unnested, all the unnested columns are "zipped up" together, with `NULL` filling in the "output" values for shorter arrays. |
+
+## Date/time functions and operators
+
+### date_trunc
+| Name | Return type | Description |
+|------|-------------|-------------|
+| date_trunc(string *precision*, timestamp *t*) | timestamp | Truncates the timestamp to the chosen *precision* ("second", "minute", "hour", "day", "week", "month", or "year"). |
+
+### date_diff
+| Name | Return type | Description |
+|------|-------------|-------------|
+| date_diff(string *precision*, timestamp *t1*, timestamp *t2*) | integer | Returns the difference between two dates, in the precision specified. *precision* is one of("second", "minute", "hour", "day", "week", "month", "quarter", "year") |
+
+<!-- QUERY below function description is unclear. Time in seconds since the Unix epoch? -->
+
+### to_timestamp
+| Name | Return type | Description |
+|------|-------------|-------------|
+| to_timestamp(numeric *n*) | timestamp | Transforms *n* into a timestamp, considering *n* as the time in seconds. |
+
+### Conditional expressions
+
+### coalesce
+| Name | Return type | Description |
+|------|-------------|-------------|
+| coalesce(expr *x*, *y*, ...) | variable | Returns the first non-null expression. |
+
+### nullif
+| Name | Return type | Description |
+|------|-------------|-------------|
+| nullif(expr *x*, expr *y*) | variable | Returns `NULL` if both arguments are equal. Otherwise, returns *x*. |
+
+## JSON functions and operators
+
+<!-- QUERY: An example of the below function would be helpful, if we're going to include the function in the docs. -->
+
+### json_extract_path_text
+
+| Name | Return type | Description |
+|------|-------------|-------------|
+| json_extract_path_text(text *json*, text *path...*) | text | Extracts the JSON sub-object in text defined by the path. Behavior is equivalent to that of the Postgres function of the same name. In the default executor, this function only works on resources tables; it is not implemented for metrics, events, or local tables. To extract nested fields in events tables, reference the entire path in double quotes as the column name. |
+
+## Timeseries and point functions
```

Comment:
Some of the verbiage on these function descriptions implies that the function can take an array of timeseries, but the parameters in the function signature don't indicate that. Please verify that these are correct.

Reply:
They only accept 1 arg, a timeseries. Not sure what implies they can take an array of timeseries?

Reply:
I had cleaned up the descriptions from Confluence, so if they look right to you here, that's enough for me -- I wrote what made sense to me based on the function name / args.